### ISSUE TITLE
Add publishing into s3 during release

### ISSUE THREAD

**Issue by Mr-Geekman** [etna-team/etna#52]
### 🚀 Feature Request

We should add publishing into s3 during release.

### Proposal

Update CI pipeline `publish/build-docs`, it should publish docs into s3 instead of Netlify. Imagine we are publishing the version `v`:
- Build documentation
- Install s3cmd
- Extract current version (see how it is done in `docker-build-and-push` job)
- Clear `v` folder
- Put documentation into `v` folder
- Clear `stable` folder
- Put documentation into `stable` folder

We are assuming that last published version is stable. If it isn't the case the developer should manually make `stable` contain the actual stable branch.

For debugging the developer could:
- Separate job into separate `yaml` file
- Activate job in pull request
- Set necessary github variables using env variables (e.g. rename them and set)

At the end you could fix `index.html` to point to the stable version.

### Additional context

Is blocked by #48.

Multiversion switch will be added in a separate task.

**Comment by d-a-bunin** [comment]
Closed by #53.
